### DISCUSSION TITLE
Out of Pegs Improvement

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#448] [Ideas]
RoughAnimator and Callipeg both have the ability to edit the out of peg's frames without resetting it. In Callipeg's case, you just click the frame that's out of pegs, tap "Edit Out of Pegs", and you don't have to reset the out of pegs. It'd be nice if ToonSquid had that ability to edit the out of pegs as you draw without resetting it.

**Comment by keiwando** [comment]
Thanks for the feedback!
